src/Card/SummaryCard/SummaryCard.tsx

```diff
@@ -304,16 +322,17 @@ function SummaryCard({
             }}
           />
         </Box>
-        <CardContent
+
+        <Box
           sx={{
-            height: height - mediaHeight - 196,
+            height: Number(height) - mediaHeight - 196,
```

Comment:
So what happens if I give this component a string height of "100%". See corresponding comment on the height type. You're just ignoring a bug that typescript is trying to tell you about here.